docs/topics/api/addons.rst

```diff
@@ -47,6 +47,7 @@ This endpoint allows you to search through public add-ons.
     :query int page: 1-based page number. Defaults to 1.
     :query int page_size: Maximum number of results to return for the requested page. Defaults to 25.
     :query string platform: Filter by :ref:`add-on platform <addon-detail-platform>` availability.
+    :query string tag: Filter by exact tag name. Multiple tag names can be specified, separated by comma(s).
```

Comment:
There are 2 types of tags, some are admin defined, some are developer defined. When you add tags as a developer, [you must separate them by commas](https://github.com/mozilla/addons-server/blob/master/src/olympia/devhub/templates/devhub/addons/edit/basic.html#L143-L146), and [the tags get split by comma as cleaning time](https://github.com/mozilla/addons-server/blob/master/src/olympia/addons/forms.py#L55), so in theory they should not contain a comma.